### ISSUE TITLE
dist in 2.1.2 broken

### ISSUE THREAD

**Issue by sky-code** [michaelfitzhavey/vue-json-tree-view#23]
After upgrade from 2.1.1 to 2.1.2 I get an error in console
```
[Vue warn]: You are using the runtime-only build of Vue where the template compiler is not available. Either pre-compile the templates into render functions, or use the compiler-included build.

found in

---> <TreeView>
```
and component not working anymore

**Comment by arvidkahl** [comment]
Hi. I have rebuilt the dist and released 2.1.3. Please tell me if it works for you again here.

**Comment by sky-code** [comment]
No, same error
![image](https://user-images.githubusercontent.com/5621185/34905632-d5581a48-f865-11e7-92e5-875a1e7c95e1.png)

**Comment by arvidkahl** [comment]
Hm, then I guess I don't really know what to do here. Is there a chance you can figure out how to build the tree component with the help of https://github.com/vuejs-templates/webpack/issues/215 and submit a pull request? That'd be great. Cause I am moving right now, barely have time to check my emails.

**Comment by sky-code** [comment]
Sorry, I have no time for this. Perhaps it is necessary to update build packages (webpack, vue-loader, etc)

**Comment by arvidkahl** [comment]
Anyone else then? :)

**Comment by jpbecotte** [comment]
Hi @arvidkahl ! Some of us are using the runtime only. In your index.js, you should define a render function. I can submit a PR if your want.

**Comment by arvidkahl** [comment]
That would be fantastic. Please do!

**Comment by GabKore** [comment]
Same issue here..

**Comment by arvidkahl** [comment]
Again, I am not actively using this any more. If anyone wants to supply a working pull request, I will merge it.

**Comment by morrislaptop** [comment]
See https://github.com/arvidkahl/vue-json-tree-view/issues/10#issuecomment-379007256

**Comment by michaelfitzhavey** [comment]
Closed as seems to be a duplicate of #10
